canvas_modules/common-canvas/src/common-properties/controls/selectcolumns/selectcolumns.jsx

```diff
@@ -145,6 +146,8 @@ class SelectColumns extends AbstractTable {
 				<ValidationMessage state={this.props.state} messageInfo={this.props.messageInfo} />
 			</div>
 		);
+		const emptyTableButtonClickHandler = this.addOnClick.bind(this, this.props.propertyId);
```

Comment:
I'm not sure if these 2 lines should be in the render method.  Shouldn't these be part of the constructor or componentDidMount?